lib/application/index.js

```diff
@@ -4,6 +4,9 @@
  * configuration and rendering.
  */
 
+// Local
+var Promise = require('../polyfills/promise');
```

Comment:
That's true, but the extra bytes we're pushing, after minification and gzipping, only add 1131, which is pretty insignificant. I can see the point, though, and can remove these 2 import lines, in which case, no documentation will need to be updated.